dashboard/app/views/levels/_blockly.html.haml

```diff
@@ -19,4 +19,4 @@
     appOptions.locale = '#{js_locale}';
     //]]>
 
-  %script{src: '/shared/js/shared.js'}
+  %script{src: "/shared/js/shared#{'.min' unless Rails.configuration.pretty_sharedjs}.js"}
```

Comment:
hm, may be possible to override <a href="http://apidock.com/rails/v4.2.1/ActionView/Helpers/AssetTagHelper/javascript_include_tag">javascript_link_tag</a> to do the right thing when it sees a path that starts with /shared (and stick with the default behaviour when it does not).

Either way I think ApplicationHelper is where this stuff usually goes.